<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="addBlockPushButton">
        <property name="text">
         <string>Add new block</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="aboutAuthorPushButton">
        <property name="text">
         <string>About author</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;BlockChain&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="authorizeBlockPushButton1">
            <property name="text">
             <string>Authorize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addTransactionPushButton1">
            <property name="text">
             <string>Add transaction</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="addressLabel1">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="addressLineEdit1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dateLabel1">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dateLineEdit1"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="hashLabel1">
            <property name="text">
             <string>Block hash</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="hashLineEdit1"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="numberLabel1">
            <property name="text">
             <string>Block number</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="numberLineEdit1"/>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="transactionsPushButton1">
            <property name="text">
             <string>Transactions</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="authorizeBlockPushButton2">
            <property name="text">
             <string>Authorize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addTransactionPushButton2">
            <property name="text">
             <string>Add transaction</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="addressLabel2">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="addressLineEdit2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dateLabel2">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dateLineEdit2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="hashLabel2">
            <property name="text">
             <string>Block hash</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="hashLineEdit2"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="numberLabel2">
            <property name="text">
             <string>Block number</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="numberLineEdit2"/>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="transactionsPushButton2">
            <property name="text">
             <string>Transactions</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="authorizeBlockPushButton3">
            <property name="text">
             <string>Authorize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addTransactionPushButton3">
            <property name="text">
             <string>Add transaction</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="addressLabel3">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="addressLineEdit3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dateLabel3">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dateLineEdit3"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="hashLabel3">
            <property name="text">
             <string>Block hash</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="hashLineEdit3"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="numberLabel3">
            <property name="text">
             <string>Block number</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="numberLineEdit3"/>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="transactionsPushButton3">
            <property name="text">
             <string>Transactions</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevBlockPushButton">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="gotoPushButton">
        <property name="text">
         <string>Go to block number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gotoLineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="nextBlockPushButton">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
